roles/girder-service-install/meta/main.yml

```diff
@@ -8,6 +8,5 @@ dependencies:
     group: "{{ girder_group }}"
     description: Girder Data Management Platform -- Web Service
     command: >
-        bash -c "cd {{ girder_install_root }} && python -m girder"
+        bash -c ". {{ girder_venv_root }}/bin/activate && cd {{ girder_install_root }} && python -m girder"
```

Comment:
```&&``` ensures that additional commands are not run if the first command fails.  this keeps errors clean and prevents unexpected side effects.